### PR TITLE
Add some new pages to addon docs

### PR DESCRIPTION
**Pull request by Hexagonl**
I made a draft PR for now because I'd like for @Hans5958 to double check what I've made so far and ensure I'm on the right path. I'll continue making more pages and put the commits for those in this draft PR, to reduce cluttering the repository's PR history.

I'll also propose some guideline additions to the [addon docs wiki](https://github.com/ScratchAddons/website-v2/wiki/Addon-Docs) in a separate issue/PR, which will be based off of stuff I've encountered while making the 60fps addon doc.

### Current added addon docs
- 60fps
- Paint costume by default

### CODE REVIEW

**Comment by Hans5958** [comment]
Thank you! I guess I have few suggestions.

1. Instead of "[Issue #6860](https://github.com/ScratchAddons/ScratchAddons/issues/6860)", I prefer "[#6860](https://github.com/ScratchAddons/ScratchAddons/issues/6860)". For that kind of usage I would prefer this instead.

   > - The addon might be marked as dangerous to curb projects that require this addon on the Scratch website ([#6860](https://github.com/ScratchAddons/ScratchAddons/issues/6860)).

   Although I think "issue #6860" (with a little I) should be used just so the general users know that it refers to an issue, but I'm not sure if this is really needed.

2. I think in TurboWarp, it is mentioned as "TurboWarp Addons", with a capital Addons.
3. I would want to see the usage of the footnotes, especially the Trivia which I think you should cite. It works similary like GitHub. For example, like this[^some-cite-name] and this.[^1]

[^1]: It doesn't have to be a number, but you can.
[^some-cite-name]: Hello! This should still appear first even though it is written last on Markdown.

   > ```
   > For example, like this[^some-cite-name] and this.[^1]
   >
   > [^1]: It doesn't have to be a number, but you can.
   > [^some-cite-name]: Hello! This should still appear first even though it is written last on Markdown.
   > ```

**Comment by WorldLanguages** [comment]
Nice work!

File: content/addons/60fps.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+**Higher project framerate mode** is an addon that allows customization of the project's run speed to be faster than the default of 30 FPS. This has the effect of making the project appear to run smoother and at a faster rate relative to the custom FPS value; therefore, the default value of 60 FPS will essentially make the project run twice as fast.
+
+The feature can be quickly toggled on/off by holding `alt` and clicking the green flag. The flag will turn blue and a yellow fast-forward icon will appear over it, indicating that the project is running at a faster speed.
+
+## Features
+
+- The addon's functionality is only enabled whenever the user toggles it on by holding `alt` and clicking the green flag. The addon's functionality is toggled off every time the page is opened/refreshed.
+- The addon works in both the project page and the editor.
+- By default, the addon (when toggled on) sets the project's framerate to 60 FPS. This value can be changed in the addon's settings to an integer ranging from 31 to 240.
+- To prevent abuse and cheating in cloud-based multiplayer games, the addon does not work whenever there are cloud variables in a shared project.
```
Comment (WorldLanguages) [inline content/addons/60fps.md:15]: Really?

Comment (WorldLanguages) [reply content/addons/60fps.md:15]: I don't think we check whether there are cloud variables or not.

**Comment by WorldLanguages** [comment]
@Hans5958 Please review again if you have some time.

I don't think we should be strict here, as long as there's not any false statements being added to the docs, any page is better than no page

File: content/addons/paint-by-default.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+{{< docs/stub-section >}}
+
+## Gallery
+
+{{< docs/stub-section >}}
+
+## Related
+
+- Original pull request: [PR #3199](https://github.com/ScratchAddons/ScratchAddons/pull/3199)
```
Comment (Hans5958) [inline content/addons/paint-by-default.md:63]: My preference is either...

```suggestion
- [Original pull request](https://github.com/ScratchAddons/ScratchAddons/pull/3199)
```

...or...


```suggestion
- [Original pull request (#3199)](https://github.com/ScratchAddons/ScratchAddons/pull/3199)
```

Also, as stated before, the term "issue" and "PR" is not needed.

**Review by Hans5958 — APPROVED** [review]
To be fair, I'm pretty fine to merge this since the beginning. Problem is that this is still a draft, so it is up to him if he wants this to be merged.

File: content/addons/60fps.md (hunk 1 above)
Comment (mxmou) [reply content/addons/60fps.md:13]: > @Hexagonl Was this removed (or the statement checked?)

It hasn't been removed. The statement is definitely incorrect - there are no references to cloud variables in the addon's code.

Comment (WorldLanguages) [reply content/addons/60fps.md:13]:
```suggestion
```

**Comment by WorldLanguages** [comment]
Before merging see: https://github.com/ScratchAddons/website-v2/pull/459#discussion_r1703332505

File: content/addons/60fps.md (hunk 1 above)
Comment (Hexagonl) [reply content/addons/60fps.md:13]: > @Hexagonl Was this removed (or the statement checked?)

I verified it myself and I did remove it in my latest commit

**Comment by Hexagonl** [comment]
I fixed my earlier mistake of the cloud variables in the 60fps addon, made the requested changes, and also added an additional reference in the 60fps addon. I also marked as ready to review

**Review by WorldLanguages — APPROVED** [review]

**Pull request merged**